templates/component-pages/README.md

```diff
@@ -0,0 +1,8 @@
+# How to use the component page MDX template
+
+To add a new component page to the Salt website:
+
+1. Copy the [`component-name/`](./component-name/) folder into the [`site/docs/components/` directory](../../site/docs/components/) and rename it to your component's name in kebab case. For example `form-field/`.
+   - 💡 **Tip:** If you are using GitHub's website to edit component docs, you can press the <kbd>.</kbd> key to launch a version of VS Code in your browser and copy the directory via that.
+1. Fill in the `index.mdx` file's frontmatter (refer to the comments within that file for guidance)
```

Comment:
Are these meant to be numbered 2. and 3. respectively? (Can't seem to find the link to preview on site)

Reply:
This README doesn't appear within our site. It just for when folks are browsing our repo.

When markdown is rendered to HTML, the numbers get replaced with the correct ones automatically: https://github.com/jpmorganchase/salt-ds/tree/add-component-template/templates/component-pages